mjtest-rs/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "mjtest"
+version = "0.0.1"
+authors = [
+    "flip1995 <hello@philkrones.com>",
+    "hediet <>",
+    "joshuabach <>",
+    "problame <>",
+    "reiner-dolp <>"
+]
+# edition = "2018" https://users.rust-lang.org/t/how-to-use-proc-macro-on-rust-2018/20833/3
```

Comment:
Is this garbage or intentional?

Reply:
intentional, if someone wonders why we can't use 2018 yet